src/render/references.hbs

```diff
@@ -4,7 +4,8 @@
 <details data-key="{{citation_key}}" class=ref>
 {{!-- citation --}}
 <summary class=citation>
-<a id="{{citation_key}}">[{{citation_key}}]</a> - {{#if authors}}{{authors}} - {{/if}}{{#if title}}<cite>{{title}}</cite>.{{/if}} - {{#if pub_year}}{{pub_year}}.{{/if}}
+{{#if authors}}{{authors}} - {{/if}}{{#if title}}<cite>{{title}}</cite>.{{/if}} - {{#if pub_year}}{{pub_year}}.{{/if}} -
```

Comment:
Here, the `<a id="{{citation_key}}"></a>` as jump anchor for the citation link is missing.

Reply:
Yes. I removed it on purpose as if was maybe too much information for the user. You prefer it back?

Reply:
I think jumping from a citation to the reference is a very convenient feature, so I would prefer to have that jump anchors.

Reply:
Added back